DetoxSync/DetoxSync/Utils/_DTXTimerTrampoline.m

```diff
@@ -16,19 +16,18 @@ @implementation _DTXTimerTrampoline
 {
 	id _target;
 	SEL _sel;
-	
+
 	//NSTimer
-	__weak NSTimer* _timer;
+	NSTimer* _timer;
```

Comment:
Ensure that we still have a reference to the `_timer` object until we are done.

I'm a bit unsure as to whether this leaves us with a circular dependency between us and the timer for releasing?